docs/pages/management/export-audit-events/panther.mdx

```diff
@@ -0,0 +1,226 @@
+#       time_type string
+#       time_format %Y-%m-%dT%H:%M:%S
+#     </parse>
+#
+#     # If the number of events is high, fluentd will start failing the ingestion
+#     # with the following error message: buffer space has too many data errors.
+#     # The following configuration prevents data loss in case of a restart and
+#     # overcomes the limitations of the default fluentd buffer configuration.
+#     # This configuration is optional.
+#     # See https://docs.fluentd.org/configuration/buffer-section for more details.
+#     <buffer>
+#       @type file
+#       flush_thread_count 8
+#       flush_interval 1s
+#       chunk_limit_size 10M
+#       queue_limit_length 16
+#       retry_max_interval 30
+#       retry_forever true
+#     </buffer>
+# </source>
+<filter test.log>
+  @type record_transformer
+  enable_ruby true
+  <record>
+    time $${time.utc.strftime(\"%Y-%m-%dT%H:%M:%SZ\")}
+  </record>
+</filter>
+<match test.log>
+  @type s3
+  aws_key_id  (=clusterDefaults.aws.aws_access_key)
+  aws_sec_key  (=clusterDefaults.aws.aws_secret_access_key=)
+  s3_bucket  (=clusterDefaults.clusterName=)
+  s3_region us-west-2
+  path teleport/logs
+  <buffer>
+    @type file
+    path /var/log/fluent/buffer/s3-events
+    timekey 60
+    timekey_wait 0
+    timekey_use_utc true
+    chunk_limit_size 256m
+  </buffer>
+  time_slice_format %Y%m%d%H%M%S
+  <format>
+    @type json
+    </format>
+</match>
+<match session.*>
+  @type stdout
+</match>
+```
+
+## Step 6/6. Run the Teleport Event Handler plugin
+
+### Configure the Event Handler
+
+In this section, you will configure the Teleport Event Handler for your
+environment.
+
+(!docs/pages/includes/plugins/finish-event-handler-config.mdx!)
+
+Next, modify the configuration file as follows:
+
+(!docs/pages/includes/plugins/config-toml-teleport.mdx!)
+
+(!docs/pages/includes/plugins/machine-id-exporter-config.mdx!)
+
+### Start the Teleport Event Handler
+
+(!docs/pages/includes/start-event-handler.mdx!)
+
+The Logs view in Panter should now report your Teleport cluster events. 
+
+## Troubleshooting connection issues
+
+If the Teleport Event Handler is displaying error logs while connecting to your
+Teleport Cluster, ensure that:
+
+- The certificate the Teleport Event Handler is using to connect to your
+  Teleport cluster is not past its expiration date. This is the value of the
+  `--ttl` flag in the `tctl auth sign` command, which is 12 hours by default.
+- Ensure that in your Teleport Event Handler configuration file
+  (`teleport-event-handler.toml`), you have provided the correct host *and* port
+  for the Teleport Proxy Service.
+- Start the FluentD container prior to starting the Teleport Event Handler. The 
+  Event Handler will attempt to connect to FluentD immediately upon startup.
+
+## Next steps
+
+- Read more about
+[impersonation](../../access-controls/guides/impersonation.mdx)
+here.
+- While this guide uses the `tctl auth sign` command to issue credentials for the
```

Comment:
Should we remove this bullet since the guide includes Machine ID instructions?

Reply:
Removed